bp-esa-frontend/src/components/reportPage.js

```diff
@@ -0,0 +1,149 @@
+                    <input
+                      type="radio"
+                      className="form-check-input"
+                      name="filter"
+                      id="partner"
+                      value="partner"
+                    />
+                    Partner
+                  </label>
+                </div>
+                <div className="form-check">
+                  <label className="form-check-label">
+                    <input
+                      type="radio"
+                      className="form-check-input"
+                      name="filter"
+                      id="developer_name"
+                      value="developer_name"
+                    />
+                    Developer name
+                  </label>
+                </div>
+                <div className="form-check">
+                  <label className="form-check-label">
+                    <input
+                      type="radio"
+                      className="form-check-input"
+                      name="filter"
+                      id="status"
+                      value="status"
+                    />
+                    Status
+                  </label>
+                </div>
+                <div className="form-check">
+                  <label className="form-check-label">
+                    <input
+                      type="radio"
+                      className="form-check-input"
+                      name="filter"
+                      id="addition"
+                      value="addition"
+                    />
+                    Addition
+                  </label>
+                </div>
+                <div className="form-check">
+                  <label className="form-check-label">
+                    <input
+                      type="radio"
+                      className="form-check-input"
+                      name="filter"
+                      id="removal"
+                      value="removal"
+                    />
+                    Removal
+                  </label>
+                </div>
+                <div className="form-check">
+                  <label className="form-check-label">
+                    <input
+                      type="radio"
+                      className="form-check-input"
+                      name="filter"
+                      id="channel"
+                      value="channel"
+                    />
+                    Channel
+                  </label>
+                </div>
+              </fieldset>
+            </Col>
+            <Col sm={9}>
+              <div className="form-group">
+                <input
+                  type="search"
+                  className="form-control"
+                  placeholder="&#xF002; Search"
+                  id="search"
+                  name="search"
+                />
+              </div>
+            </Col>
+          </Row>
+        </form>
```

Comment:
I thin it would be better for these radio buttons to live in a component of their own, and have the filter criteria (and associated functions) assigned via props; so that the same component can be reused for the General page as well.

Reply:
@Bopchy Should I now create another component that will be a parent of both Report page and General page?